spectro-application-profile.tf

```diff
@@ -0,0 +1,70 @@
+locals {
+  application_profiles = toset(flatten([
```

Comment:
sets and maps are better in spectro-application-profile-common.tf for readability.

Reply:
created common config and moved variables there